### PR TITLE
implement slpp MsgServer

### PR DESCRIPTION
**Pull request by thal0x**
Still WIP, need to write tests once the Keeper is complete.

**Edit:** nvm rip it

### CODE REVIEW

File: x/slpp/keeper/msg_server.go [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+
+// RegisterAVS does something..
+func (m *msgServer) RegisterAVS(ctx context.Context, req *types.MsgRegisterAVS) (*types.MsgRegisterAVSResponse, error) {
+	// check the validity of the message
+	if req == nil {
+		return nil, fmt.Errorf("message cannot be empty")
+	}
+
+	// dennis does everything
+	id, err := m.keeper.RegisterAVS(ctx, req)
```
Comment (g3ndo9kar9) [inline x/slpp/keeper/msg_server.go:29]: gud decision

**Review by g3ndo9kar9 — APPROVED** [review]
kobe 🔥

File: proto/cosmwasm/slpp/v1/slpp.proto [hunk 2]
```diff
@@ -9,16 +9,18 @@ service Msg {
 
 // AVS represents the state-ful information stored per AVS
 message AVS {
-    string contract_address = 1;
+  bytes contract_bin = 1;
```
Comment (NotJeremyLiu) [inline proto/cosmwasm/slpp/v1/slpp.proto:14]: Feel like the identifier here should just be code id, and the code id in the wasm module is what stores the bytes

Comment (thal0x) [reply proto/cosmwasm/slpp/v1/slpp.proto:14]: I'm a little unsure of where we left off on the conversation, but my understanding was a contract could potentially be associated with more than one AVS

**Pull request merged**